projects/lights_control/inc/lights_config.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "gpio.h"
+
+typedef enum {
+  LIGHTS_CONFIG_BOARD_TYPE_FRONT = 0,
+  LIGHTS_CONFIG_BOARD_TYPE_REAR,
+  NUM_LIGHTS_CONFIG_BOARDS
+} LightsConfigBoardType;
+
+typedef struct LightsConfig {
+  const LightsConfigBoardType board_type;
+  const GPIOAddress *addresses;
+  const uint8_t num_addresses;
+  const GPIOSettings *gpio_settings_out;
```

Comment:
Why bother with this? Aren't they just the same? Can't the settings just be in the IO module itself?

Reply:
yeah what was I thinking 😕